docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,297 @@
+```
+
+All of the nodes are sources and replicas. DC1 is a replica of DC2 and DC3. DC2 and DC3 are replicas of DC1.
+
+```text
++----------------+---------------+
+| clusterReplica | clusterSource |
++----------------+---------------+
+| DC1            | DC2           |
+| DC1            | DC3           |
+| DC2            | DC1           |
+| DC3            | DC1           |
++----------------+---------------+
+```
+
+The example of the minimal MySQL configuration file for DC1 that is common for all three DC1 nodes:
+
+```text
+[mysqld]
+# General galera reqs
+default_storage_engine=InnoDB
+innodb_autoinc_lock_mode=2
+
+# Replication settings
+binlog_format=ROW
+server-id=1
+log-bin=mysql-bin
+log_replica_updates
+expire_logs_days=7
+gtid_mode = ON
+enforce_gtid_consistency=ON
+source_info_repository = TABLE
+relay_log_info_repository = TABLE
+skip-replica-start
+    
+# Galera configuration
+wsrep_provider=/usr/lib/galera3/libgalera_smm.so
+wsrep_cluster_address=gcomm://10.0.4.160,10.0.4.162,10.0.4.163
+wsrep_replica_threads= 2
+wsrep_log_conflicts
+wsrep_cluster_name=DC1
+pxc_strict_mode=ENFORCING
+wsrep_sst_method=xtrabackup-v2
+wsrep_sst_auth="root:root"
+```
+
+All nodes have the same `server-id` value. In a multi-source replication topology, the replicas require the `source_info_repository` and `relay_log_info_repository` values to be `TABLE`. These settings cannot be `FILE`.
+
+### Configuration steps
+
+1. Bootstrap the cluster on DC1-1 node by running the following commands as root or using sudo:
+
+    ```bash
+    [root@DC1-1 ~]# /etc/init.d/mysql stop 
+    [root@DC1-1 ~]# /etc/init.d/mysql bootstrap-pxc
+    ```
+
+2. On DC1-2 node, stop MySQL, delete the content of the datadir to force the SST, and start MySQL. Run the following commands as root or using sudo:
+
+    ```bash
+    [root@DC1-2 ~]# /etc/init.d/mysql stop
+    [root@DC1-2 ~]# rm -rf /var/lib/mysql/*
+    [root@DC1-2 ~]# /etc/init.d/mysql start
+    ```
+
+3. Once the SST of DC1-2 node is completed, proceed on DC1-3. Run the following commands as root or using sudo:
+
+    ```bash
+    [root@DC1-3 ~]# /etc/init.d/mysql stop
+    [root@DC1-3 ~]# rm -rf /var/lib/mysql/*
+    [root@DC1-3 ~]# /etc/init.d/mysql start
+    ```
+
+4. The DC1 cluster uses a single [GTID](../glossary.md#gtid) sequence. To make sure `GTID_PURGED` is the same, run the following commands on DC1-1, DC1-2, and DC1-3 nodes:
+
+    ```bash
+    mysql> flush logs;
+    mysql> purge source logs to 'mysql-bin.000003';
+    ```
+
+    `mysql-bin.000003` is the last file returned from the show source logs. 
+
+5. Adjust the aformentioned steps to set up the DC2 and DC3 clusters accordingly.
+    
+6. Add privileges for replication between all nodes only for DC1: 
+
+    ```bash
+    GRANT REPLICATION CLIENT, REPLICATION REPLICA ON *.* TO 'repl'@'%' identified by 'replpass';
+    ```
+
+7. Add the replication tables, the required links, and assign the weights. For example, we add the tables to DC1: 
```

Comment:
Since you made them separate steps, you can remove the "required links and assign the weights". Those are not part of this step.

Reply:
Updated